src/PrestaShopBundle/Controller/Admin/Sell/Order/InvoicesController.php

```diff
@@ -121,6 +132,138 @@ public function processAction(Request $request)
         return $this->redirectToRoute('admin_order_invoices');
     }
 
+    /**
+     * @return array
+     *
+     * @throws FieldNotFoundException
+     *
+     * @var InvalidConfigurationDataErrorCollection
+     */
+    private function getErrorMessages(InvalidConfigurationDataErrorCollection $errors): array
+    {
+        $messages = [];
+
+        foreach ($errors as $error) {
+            $messages[] = $this->getErrorMessage($error);
+        }
+
+        return $messages;
+    }
+
+    /**
+     * @param InvalidConfigurationDataError $error
+     *
+     * @return string
+     *
+     * @throws FieldNotFoundException
+     */
+    private function getErrorMessage(InvalidConfigurationDataError $error): string
+    {
+        switch ($error->getErrorCode()) {
+            case InvoicesByDateDataProvider::ERROR_INVALID_DATE_TO:
+            case InvoicesByDateDataProvider::ERROR_INVALID_DATE_FROM:
+            return $this->trans(
+                    'Invalid "%s" date.',
+                    'Admin.Orderscustomers.Notification',
+                    [
+                        $this->getFieldLabel($error->getFieldName()),
+                    ]
+                );
+            case InvoicesByDateDataProvider::ERROR_NO_INVOICES_FOUND:
+                return $this->trans(
+                    'No invoice has been found for this period.',
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvoicesByStatusDataProvider::ERROR_NO_ORDER_STATE_SELECTED:
+                return $this->trans(
+                    'You must select at least one order status.',
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvoiceByStatusFormHandler::ERROR_NO_INVOICES_FOUND_FOR_STATUS:
+                return $this->trans(
+                    'No invoice has been found for this status.',
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvoiceOptionsDataProvider::ERROR_INCORRECT_INVOICE_NUMBER:
+                return $this->trans(
+                    'Invoice number must be greater than the last invoice number, or 0 if you want to keep the current number.',
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvoiceOptionsDataProvider::ERROR_CONTAINS_HTML_TAGS:
+                if ($error->getLanguageId()) {
+                    $langRepository = $this->get('prestashop.core.admin.lang.repository');
+                    $lang = $langRepository->findOneBy(['id' => $error->getLanguageId()]);
+
+                    return $this->trans(
+                        'Field "%s" in language "%s" is invalid. Field must not contain HTML tags.',
```

Comment:
```suggestion
                        'The "%s" in "%s" is invalid. HTML tags are not allowed.',
```

What do you think of my suggestion? I feel like we would better say 'The Footer text in English is invalid.' rather than 'Field Footer text in language English is invalid."

Reply:
Just want to make sure it only has to do with 'Footer text' and 'Legal free text'. If yes 'The "%s"' is working.

Reply:
Makes perfect sense, thanks! Added it.